Makefile

```diff
@@ -23,6 +23,17 @@ GOPROXY=direct
 GOPATH=$(shell go env GOPATH)
 GOOS=$(shell go env GOOS)
 GOBIN=$(shell pwd)/bin
+GO ?=go
+deps_diff := diff --no-dereference -N
+
+# $1 - temporary directory
+define restore-deps
+	ln -s $(abspath ./) "$(1)"/current
+	cp -R -H ./ "$(1)"/updated
+	$(RM) -r "$(1)"/updated/vendor
```

Comment:
is there a reason to use variable for "rm" ?

Reply:
Not particularly. I went ahead and removed this, along with adding the `-f` flag to rm.